docs/manual/helloworld-unity-importwebrtc.md

```diff
@@ -31,27 +31,36 @@ cd /D D:\testproj
+For the native implementation `mrwebrtc.dll` things are a bit more complex. The DLL is compiled for a particular platform and architecture, in addition of the Debug or Release build config, and the correct variant needs to be used. On Windows, the Unity Editor needs a **64-bit Desktop** variant (`Debug` or `Release`); it is available from the `bin\Win32\x64\Release` folder (`Win32` is an alias for `Desktop`), and should be copied to the `Assets\Plugins\Win32\x86_64\` folder of the Unity project.
 
 ```
 cd /D D:\testproj
-xcopy D:/mr-webrtc/bin/Win32/x64/Release/Microsoft.MixedReality.WebRTC.Native.dll Assets/Plugins/Win32/x86_64/
+xcopy D:/mr-webrtc/bin/Win32/x64/Release/mrwebrtc.dll Assets/Plugins/Win32/x86_64/
 ```
 
 ## Configuring the import settings
 
- When building the Unity application for a given platform, another variant may be required. In order for the C# library to be truly platform-independent, the name of all C++ library variants is the same. This allows the C# code to reference the C++ library with [the same `DllImport` attribute path](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.dllimportattribute?view=netcore-2.1). But this also means that Unity needs to know which copy is associated with which build variant, to be able to deploy the correct one. This is done by configuring the platform associated with a DLL in the import settings in the Unity inspector:
+When building the Unity application for a given platform, another variant than the one the Unity editor uses may be required to deploy on that platform. In order for the C# library to be truly platform-independent, the name of all native implementation library variants is the same: `mrwebrtc`. This allows the C# code to reference that DLL with [the same `DllImport` attribute path](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.dllimportattribute?view=netcore-2.1). But this also means that multiple files with the same name exist in different folders, and Unity needs to know which one is associated with which build variant, to be able to deploy the correct one only. This is done by configuring the platform associated with a DLL in the import settings in the Unity inspector.
 
-![Configure the import settings for a native C++ DLL](helloworld-unity-2.png)
+> [!IMPORTANT]
+> The `mrwebrtc.dll` module was previously named `Microsoft.MixedReality.WebRTC.Native.dll` when this module was also the C++ library. Those two have now been split apart (see issue [#123](https://github.com/microsoft/MixedReality-WebRTC/issues/123), and commit [169502a](https://github.com/microsoft/MixedReality-WebRTC/commit/169502aab851b61c260831892e3eff9deeb876ae) for rename). The `Microsoft.MixedReality.WebRTC.Native` name is still in use in some NuGet packages, so depending on the installation method one or the other name are used. However the Unity project must use the `mrwebrtc.dll` name only for the C# library to find it.

```

Comment:
nit: I'd leave out commit and issue, not really relevant within the tutorial docs.

Reply:
I think it is in the sense that NuGet packages and local builds will have different file names.

Reply:
Yeah it's fine to point that out, but the exact commit isn't that useful, most people won't care about rolling back to a precise commit.

Reply:
Removed.

Reply:
Good, I'd change "previously" with "in 1.0" which seems a good tradeoff between the exact commit and not saying at all.

Also see reference at line 61.

Reply:
Fixed.